ppg/pg_stat_monitor-autobuild.groovy

```diff
@@ -230,33 +246,33 @@ pipeline {
         } //stage
         stage('Build pg_stat_monitor DEBs') {
             parallel {
-                stage('Ubuntu 18.04') {
+                stage('Ubuntu 20.04') {
                     agent {
-                        label 'min-bionic-x64'
+                        label 'min-focal-x64'
                     }
                     steps {
-                        echo "====> Build pg_stat_monitor deb on Ubuntu 18.04 PG${PG_RELEASE}"
+                        echo "====> Build pg_stat_monitor deb on Ubuntu 20.04 PG${PG_RELEASE}"
                         cleanUpWS()
                         installCli("deb")
                         unstash 'properties'
                         popArtifactFolder("source_deb/", AWS_STASH_PATH)
-                        buildStage("ubuntu:bionic", "--build_deb=1")
+                        buildStage("ubuntu:focal", "--build_deb=1")
 
                         pushArtifactFolder("deb/", AWS_STASH_PATH)
                         uploadDEBfromAWS("deb/", AWS_STASH_PATH)
                     }
                 } //stage
-                stage('Ubuntu 20.04') {
+                stage('Ubuntu 22.04') {
```

Comment:
we need both 20 and 22

Reply:
Yes we have stages for both 20.04 and 22.04. I removed bionic and focal took its place in git diff.
-              stage('Ubuntu 18.04') {
+                stage('Ubuntu 20.04') {